CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 2.18.0-rc.0 / 2020-04-24
+
+* [CHANGED] Use .UTC timezeone everywhere we use time.Unix. #7066
+* [CHANGED] Federation: It does not use remote storages anymore as designed initially. #7077
+* [CHANGED] Rules: `rule_evaluations_total` and `rule_evaluation_failures_total` have `rule_group` label now. #7094
+* [CHANGED] Moved to Go 1.14.2. #7100
+* [FEATURE] Tracing(!): Added experimental Jaeger support #7148
+* [ENHANCEMENT] TSDB: Significantly reduce how much old WAL we keep around to 3h (previously 6h). #7098
```

Comment:
is it needed to specify the hours?

Reply:
No harm, it's useful to get idea

Reply:
is the WAL size depending on parameters that can be changed? Can we instead say divided by two? WDYT?

Reply:
Fair point, Happy with that